### ISSUE TITLE
Broken in IE<9. Possibly outerHTML?

### ISSUE THREAD

**Issue by jeffhume** [kolber/audiojs#67]
I'm having trouble getting this to work in IE versions < 9. Strangely, however, it works in some situations and not others. I can't make sense of why.

Currently the player doesn't work and throws up a generic javascript error at http://iplawpodcast.jurisdiction.com. Strangely, however, it does work at http://iplawpodcast.jurisdiction.com/testing/index.html

It uses the exact same audio.min.js file. I modified it to take out some of the styles for my own purposes but when I use the original file directly downloaded from here it has the same problem.

I can't figure out why it would work on this one page but not the other. The page it doesn't work on is a wordpress page, but it has a very limited template. I'm not even loading in any other javascript except for an html5 shiv. When I remove that though the error persists, so it doesn't appear to be a conflict with that.

It brings up a generic IE javascript error alert saying the error is at line 10 character 446. At that point in the file the code is:

b.outerHTML=d.outerHTML;d=document.getElementById(c)}else{d.appendChild(e);

I found something else referencing outerHTML issues (https://github.com/kolber/audiojs/pull/33), but it appears to be fixed so I'm not sure if this problem is related or not.

Any help or advice would be much appreciated. I apologize if it's something straightforward, but I can't figure it out.

Thanks very much. It's a great player.

**Comment by listao** [comment]
Yeah, I see to be having the same issue. For me IE8 keeps reporting a the error `'parentNode' is null or not an object`. In the uncompressed version, it is Line 584, character 5. Having traced it through the calls, it seems for me I'm getting that issue at inside the `createPlayer()` function call: 

```
// Tracing this out in IE8 debugger  reveals it returns a length of 0...
return wrapper.getElementsByTagName('audio')[0];
```

It can't seem to find the `<audio>` element. Weird ,that like other users, this works fine when I have the player just out in the open, but when I have the string block created then later initialize the player, it throws this issue.
